core-common/src/main/java/org/zalando/nakadi/service/subscription/zk/ZkSubscriptionClient.java

```diff
@@ -202,21 +203,18 @@ public Partition[] getPartitions() {
         }
 
         public Topology withUpdatedPartitions(final Partition[] partitions) {
```

Comment:
```suggestion
        public Topology withUpdatedPartitions(final Partition[] newPartitions) {
```
It's confusing when the parameter hides the class field.